Database/corrections.lua

```diff
@@ -326,6 +326,9 @@ QuestieCorrections.questFixes = {
     [1085] = {
         [QuestieDB.questKeys.preQuestSingle] = {1070},
     },
+    [4496] = {
+        [QuestieDB.questKeys.preQuestGroup] = {4493, 4494},
```

Comment:
```suggestion
        [QuestieDB.questKeys.preQuestSingle] = {4493,4494},
```

I've updated our requirement logic so that the preQuestSingle field is actually used for what it is supposed to.